robber/matchers/exception.py

```diff
@@ -0,0 +1,33 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
```

Comment:
@oyster Yes, it's PEP8 convention
https://www.python.org/dev/peps/pep-0008/#id21
> Surround top-level function and class definitions with two blank lines.